windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+
+At this point Certbot is ready to run.
+
+## Using Certbot
+
+Certbot is a command line interface with no UI. To interact with it you will need a command line: the classic one (`cmd.exe`) and Powershell aor both supported. You can also use Bash for Windows (available if you installed GIT for Windows for instance) if you prefer.
+
+The command line needs to be executed under **administrative privileges**. Otherwise Certbot will fail to run.
+
+Once the command line is opened, simply write the following command to get the Certbot help:
+```
+certbot --help
+```
+
+Indeed the Certbot installer set up the `PATH` to expose the command line interface `certbot`. So you can run it from any folder, or even include it in your scripts.
+
+In respect with the supported features as described in the _Supported features_ section, you can check the inline help or https://certbot.eff.org/docs/ to check how to use a specific feature in Certbot.
+
+## Configuration
+
+Certbot works essentially with a configuration folder, where all the configuration files and generated certificates are available.
+
+On Windows, this folder is `C:\Certbot`. This can be changed using the `--config-dir` flag on the Certbot CLI.
+
+Within this folder you will find the following folders:
+* `C:\Certbot\live`: contains current versions of issued certificates and keys
+* `C:\Certbot\archive`: contains all current and previous versions of issued certificates and keys
+* `C:\Certificate\logs`: contains Certbot activity logs
+
+Doing backups of `C:\Certbot` folder is everything required to save your Certbot installation.
+
+## Certificates renewals
+
+Renewals are supported by Certbot for Windows and are even already configured. Indeed the Certbot installer created a scheduled task during the installation to execute `certbot renew` twice a day.
+
+All certificates created with Certbot for Windows are then automatically renewed out-of-the-box, providing that required environment to renew is available (eg. credentials, running web server ...).
+
+## A note on manual hooks
+
+As explained by the [manual plugin documentation](https://certbot.eff.org/docs/using.html#manual), you may need to setup authentication and cleanup hooks to make the `manual` plugin work as expected, in particular with `DNS-01` challenges.
+
+Similarly to Linux, Certbot for Windows supports any file executable for this platform, if it is available in the `PATH`. It can be `.bat`, `.ps` or even executable binaries like `.exe`.
+
+## A note on files security
+
+Assets generated by Certbot, in particular keys, are critical in term of security and should never been exposed to unauthorized accesses. Certbot for Windows leverages DACLs implemented by the NTFS filesystem to enforce this control: keys and sensible materials are readable by default only by the account that runs Certbot, the `Administrators` group and `SYSTEM`.
```

Comment:
sensitive materials

("sensible" is a false friend of French "sensible" in this sense)